src/Symfony/Component/Form/Tests/Extension/Validator/Constraints/FormValidatorTest.php

```diff
@@ -149,11 +127,11 @@ public function testValidateIfChildWithValidConstraint()
         $this->assertNoViolation();
     }
 
-    public function testDontValidateIfParentWithoutCascadeValidation()
+    public function testDontValidateIfParentWithoutValidConstraint()
```

Comment:
From my understanding it looks like it is not duplicated as we have the related tests

- testValidateConstraintsEvenIfNoValidConstraint
- testValidateIfChildWithValidConstraint
- testDontValidateIfParentWithoutValidConstraint